src/psyclone/domain/lfric/arg_ordering.py

```diff
@@ -447,6 +451,21 @@ def stencil_2d_unknown_extent(self, arg, var_accesses=None):
 
         '''
 
+    # @abc.abstractmethod
+    def stencil_2d_max_extent(self, arg, var_accesses=None):
+        '''Add 2D stencil information to the argument list associated with the
+        argument 'arg' if the maximum branch extent is unknown. If supplied it
```

Comment:
This wouldn't be specified in the metadata as it's calculated from the stencil extent passed in from the Algorithm (or which could be in the metadata when the functionality is turned back on).

Reply:
So should this say "if the stencil extent (from which it is calculated) is passed from the Algorithm layer rather than being specified in kernel metadata"?